### PR TITLE
[Remove Stream/Optional usage] LPS-173787 module text-localizer

### PR DESCRIPTION
**Pull request by JanisZhang**
Please note this module is in "Total usages". Thanks!

Ticket: https://issues.liferay.com/browse/LPS-173787

### CODE REVIEW

**Comment by dantewang** [comment]
@JanisZhang Is this one present at the "Total Usages" page in our google sheet?

File: modules/apps/text-localizer/text-localizer-taglib/src/main/java/com/liferay/text/localizer/taglib/internal/address/JPAddressTextLocalizer.java [hunk 1]
```diff
@@ -37,23 +35,20 @@ public class JPAddressTextLocalizer implements AddressTextLocalizer {
...
 
 		Address escapedAddress = address.toEscapedModel();
 
 		String city = escapedAddress.getCity();
 
 		boolean hasCity = Validator.isNotNull(city);
 
-		Optional<String> regionNameOptional = AddressUtil.getRegionNameOptional(
-			address);
+		String regionName = AddressUtil.getRegionName(address);
```
Comment (dantewang) [inline modules/apps/text-localizer/text-localizer-taglib/src/main/java/com/liferay/text/localizer/taglib/internal/address/JPAddressTextLocalizer.java:51]: @JanisZhang Could you SF it to move this variable down before line 61? It's not used until that line but declared 10 lines earlier.

**Review by dantewang — CHANGES_REQUESTED** [review]

**Comment by JanisZhang** [comment]
> @JanisZhang Is this one present at the "Total Usages" page in our google sheet?

Hi @dantewang, Yes. Since the modules assigned to me are merged or in review core/Brian, so I'm working on this part.

File: modules/apps/text-localizer/text-localizer-taglib/src/main/java/com/liferay/text/localizer/taglib/internal/address/JPAddressTextLocalizer.java (hunk 1 above)
Comment (JanisZhang) [reply modules/apps/text-localizer/text-localizer-taglib/src/main/java/com/liferay/text/localizer/taglib/internal/address/JPAddressTextLocalizer.java:51]: sure Thanks!

**Comment by JanisZhang** [comment]
Hi @dantewang , force-pushed. Thanks. I append 1 SF commit.

**Comment by dantewang** [comment]
ci:forward

**Pull request closed without merging**